base/models/models.go

```diff
@@ -69,6 +70,15 @@ type AdvisoryAccountData struct {
 func (AdvisoryAccountData) TableName() string {
 	return "advisory_account_data"
 }
+type SystemAdvisoriesSlice []SystemAdvisories
+
+func (this SystemAdvisoriesSlice) ToInterfaceSlice() []interface{} {
+	var res []interface{}
```

Comment:
As we know source array size, could we allocate result array in the beginning?

Reply:
The same optimization could be done in the `OnConflictUpdateMulti` method.

Reply:
Fixed, thanks